org.eclipse.xtext/src/org/eclipse/xtext/validation/AbstractDeclarativeValidator.java

```diff
@@ -46,8 +46,14 @@
  * }
  * </pre>
  * 
+ * <p>
+ * By default {@link NullPointerException}s occurring in the invocation of validation code are swallowed. 
```

Comment:
Nitpic: `{@ link NullPointerException NullPointerExceptions}` instead of `{@link NullPointerException}s`.

Reply:
will change that and then merge